ext/pcntl/pcntl.c

```diff
@@ -1551,6 +1560,47 @@ PHP_FUNCTION(pcntl_rfork)
 #endif
 /* }}} */
 
+#ifdef HAVE_FORKX
+/* {{{ proto bool pcntl_forkx(int flags)
+   More elaborated version of fork with few settings. */
```

Comment:
In this case, because only the two constants added here are supported as flags, it will be better to give a slightly more detailed description that may be copied for the manual/doc updates.

Reply:
fair point indeed.